Titanium.Web.Proxy/RequestHandler.cs

```diff
@@ -271,21 +259,17 @@ partial class ProxyServer
         }
 
 
-        private async Task<bool> HandleHttpSessionRequestInternal(TcpConnection connection,
-            SessionEventArgs args, bool closeConnection)
+        private async Task<bool> HandleHttpSessionRequestInternal(TcpConnection connection, SessionEventArgs args, bool closeConnection)
         {
+            bool dispose = true;

```

Comment:
The other methods have "disposed" flag, this is "dispose" which means that Dispose should be called. So this is not the return value, which is true when the streams were disposed.
I'll try to make it similar to the other methods, please wait for the next commit.